tests/Unit/GeneratorTestCase.php

```diff
@@ -265,6 +265,31 @@ public function can_call_route_and_generate_response()
         ], json_decode($parsed['response'], true));
     }
 
+    /** @test */
+    public function can_parse_response_file_tag()
+    {
+        // Create a file
+        $fp = fopen(storage_path('test.json'), 'w');
```

Comment:
Good test. But I'd rather we use a fixture here, rather than generating a file.

Reply:
done